lib/yapople.js

```diff
@@ -158,6 +158,7 @@ function onData(data) {
     if (typeof this.flow !== 'undefined') { // for first and all next data chunks
         this.flow = Buffer.concat([this.flow, data]); // append chunk to buffer
 
+        if (this._command.cmd === state.RETR && this.flow.slice(-5).toString() !== '\r\n.\r\n') return;
         if (
             this.flow.slice(this.flow.length - 3).toString() === '.\r\n' ||
```

Comment:
Maybe move the condition to the next line? Smth like this:
```js
if (
  this.flow.slice(this.flow.length - 5).toString() === '\r\n.\r\n' || 
  sData.substr(-5) === '\r\n.\r\n'
)
```
Because, at line [165-166](https://github.com/agsh/yapople/pull/32/files#diff-c7da1b7f0ac7187251e8430d7fae01459939a2d656248532e0c10a7a743dc798R166) I remove five symbols, not three at the end (I don't remember why I did this :smile:)
And I think that `\r\n.\r\n` in the specification also should work for `TOP` and `LIST` commands